tests/storage_file

```diff
@@ -0,0 +1 @@
+{"_default": {"1": {"url": "http://master.tus.io/files/foo_bar", "key": "f3e5fa051d458fb7f22eaf03749c8272"}}}
```

Comment:
Is this file intended to be committed?

Reply:
yes it is. It's used to test the resumability feature. 🙂